EleksTubeHAX/GLOBAL_DEFINES.h

```diff
@@ -11,8 +11,16 @@
 #include <stdint.h>
 #include <Arduino.h>
 
-// ************ General config *********************
-//#define HARDWARE_SI_HAI_CLOCK  // comment for EleksTube original clock; uncomment only for the fake chinese copy of the clock
+
+#include "SECRETS.h"
+//// SECRETS.h must be created ---- see secretsexample_.h

```

Comment:
Its a small thing, but change this to match the case of the actual file.
```suggestion
//// SECRETS.h must be created ---- see SECRETESexample.h

```